WebAPI/DAL/Repositories/UserRepository.cs

```diff
@@ -5,14 +5,18 @@
 using System.Threading.Tasks;
 using DAL.DataContext;
 using DAL.Entities;
+using DAL.Models;
 
 namespace DAL.Repositories
 {
     public class UserRepository : GenericRepository<User>, IUserRepository
     {
         public UserRepository(ApplicationContext context) : base(context)
         {
-
+        }
+        public User FindUserByLoginModel(LoginModel loginModel)
+        {
+            return appContext.Users.FirstOrDefault(u => u.UserName == loginModel.UserName && u.Password == loginModel.Password);
```

Comment:
probably passwords should be saved more secure than plain text, but lets keep it as is for now